HC/WrapperGen/WrapperGen.cpp

```diff
@@ -521,15 +534,13 @@ struct StringFinder
           out << "  static accelerator_view av = accelerator().get_default_view();\n";
           out << "  _lp.av = &av;\n";
           out << "}\n\n";
-          out << "completion_future cf = parallel_for_each(*(_lp.av),extent<3>(_lp.gridDim.z*_lp.groupDim.z,_lp.gridDim.y*_lp.groupDim.y,_lp.gridDim.x*_lp.groupDim.x).tile_with_dynamic(_lp.groupDim.z, _lp.groupDim.y, _lp.groupDim.x, _lp.groupMemBytes), \n"
+          out << "completion_future cf = parallel_for_each(*(_lp.av),extent<3>(_lp.grid_dim.z*_lp.group_dim.z,_lp.grid_dim.y*_lp.group_dim.y,_lp.grid_dim.x*_lp.group_dim.x).tile_with_dynamic(_lp.group_dim.z, _lp.group_dim.y, _lp.group_dim.x, _lp.dynamic_group_mem_bytes), \n"
               << func->getFunctorName()
               << "(";
           func->printArgsAsArguments(out);
           out << "));\n\n"
               << "if(_lp.cf)\n"
               << "  *(_lp.cf) = cf;\n"
-              << "else\n"
-              << "  cf.wait();\n"
```

Comment:
this make grid_launch kernels truly async, no matter a completion_future is supplied. I like having a consistent semantic.